src/packages/@ncigdc/modern_components/AffectedCasesTable/AffectedCasesTable.js

```diff
@@ -362,11 +362,13 @@ const Component = compose(
                     gender: c.demographic ? c.demographic.gender : '',
                     age_at_diagnosis: ageDisplay(diagnosis.age_at_diagnosis),
                     tumor_stage: diagnosis.tumor_stage,
-                    days_to_last_follow_up: (diagnosis.days_to_last_follow_up ||
-                      0)
-                      .toLocaleString(),
-                    days_to_death: (diagnosis.days_to_death || 0)
-                      .toLocaleString(),
+                    days_to_last_follow_up: (diagnosis.days_to_last_follow_up !=
```

Comment:
agreed, not obvious to readers that the intention is to take advantage of a js quirk 
would recommend either `_.isNil` or a comment saying `// intentional non-strict check to allow for null or undefined`